README.md

```diff
@@ -41,7 +41,7 @@ Finally, create the following `scripts` in your package.json:
 ```json
 {
   "scripts": {
-    "build": "webpack --config webpack.config.js",
+    "build": "webpack --config webpack.config.js --analyze=static",
```

Comment:
see https://code.canopy.ninja/snippets/11 -- this makes it possible to get bundle analyzer data into the gitlab project badge